build-files/build-image.sh-ppc64le

```diff
@@ -25,17 +32,19 @@ setup_dotnet_sdk() {
 patch_runner() {
   echo "Patching runner"
   cd /tmp
-  git clone -q https://github.com/actions/runner
+  git clone -q ${RUNNERREPO}
   cd runner
   git checkout $(git describe --tags $(git rev-list --tags --max-count=1))
-  git apply /home/ubuntu/runner-ppc64le.patch
+  #git apply /home/ubuntu/runner-${ARCH}.patch
+  patch -p 1 -i /home/ubuntu/runner-${ARCH}.patch
   
   #TODO: Check if patch is applied correctly
 }
 
 build_runner() {
   echo "Building runner binary"
   cd src
+  bash
```

Comment:
I didn't understand why this was added - are we not in bash already for execution?  Mostly a question for Neal/Anup, I just wasn't clear on why it is needed, not disputing it.

Reply:
Debug hangover that needs to be removed.

Reply:
Removed and squashed commit pushed.